_src/static/sass/utilities/_variables.scss

```diff
@@ -193,34 +193,31 @@ $font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace
 // typography: config - scale
 $modular-scale-ratio: $perfect-fourth;
 
-$font-scale-small: 16;
-$font-scale-medium: 18;
+$font-scale-small: 14;
+$font-scale-medium: 16;
 $font-scale-large: 18;
 $font-scale-base: $font-scale-medium;
 
-// typography: sizes
+// typography: sizes (pixels)
 $font-sizes: (
-    base:           $font-scale-base,
-    mid-small:      modular-scale(-1, $font-scale-base),
-    small:          modular-scale(-2, $font-scale-base),
-    x-small:        modular-scale(-3, $font-scale-base),
-    xx-small:       modular-scale(-4, $font-scale-base),
-    mid-large:      modular-scale(1, $font-scale-base),
-    large:          modular-scale(2, $font-scale-base),
-    x-large:        modular-scale(3, $font-scale-base),
-    xx-large:       modular-scale(4, $font-scale-base)
+    xx-large:       43,
+    x-large:        32,
+    large:          24,
+    mid-large:      21,
+    mid-small:      18,
+    base:           $font-scale-base
```

Comment:
base should be 18. This also needs to be changed in https://github.com/edx/ux-pattern-library/blob/clrux/elements-headings/_src%2Fstatic%2Fsass%2Futilities%2F_variables.scss#L18.